packages/liveblocks-client/src/room.ts

```diff
@@ -1233,7 +1236,7 @@ See v0.13 release notes for more information.
   }
 
   function broadcastEvent(
-    event: any,
+    event: fixme,
```

Comment:
Here, the type of this param is known. I haven't figured out what it is exactly (yet), but in the meantime, let's at least not disable type checking on it. Hence the `fixme` type (which is aliased to `unknown`). It signifies that I will come back to this later, and add a proper type annotation. In the meantime, though, let's avoid the `any` here.